packages/playground/website/project.json

```diff
@@ -7,7 +7,9 @@
 	"targets": {
 		"build": {
 			"executor": "nx:noop",
-			"dependsOn": ["build:wasm-wordpress-net", "build:playground-puzzle"]
+			"dependsOn": [
+				"build:wasm-wordpress-net"
```

Comment:
Unrelated change. I noticed that the `"build:playground-puzzle"` dependency was still here, but we don't need it anymore.